src/components/App/App.js

```diff
@@ -12,36 +12,44 @@ class App extends Component {
     this.state = {
       allMovies: [],
       selectedMovie: null,
-      error: null
+      error: null,
```

Comment:
I know you've already worked through this.  Let me know if anything is still unclear about using empty data states vs null.